### PR TITLE
Update frontstage variables to point at services directly

### PR DESCRIPTION
**Pull request by jcox-dev**
This is for the changes in frontstage to be merged in by https://github.com/ONSdigital/ras-frontstage/pull/333.

### CODE REVIEW

**Comment by jcox-dev** [comment]
[[TBL] Have frontstage call the other services directly](https://trello.com/c/rxYfcnBR/113-tbl-have-frontstage-call-the-other-services-directly)

**Comment by jcox-dev** [comment]
@benjefferies this also removes frontstage-api.

**Comment by jcox-dev** [comment]
@tej99 I've pulled in the commits from master.

File: .env [hunk 1]
```diff
@@ -99,21 +99,16 @@ SECURE_MESSAGE_HOST=secure-message
```
Comment (tej99) [inline .env]: `COLL_INST_PASSWORD=password` needs to be changed to `COLL_INST_PASSWORD=secret` on line 94. This seems to fix all the failing acceptance tests when running frontstage in docker.

**Pull request merged**